packages/api/tests/controllers/nomination.test.js

```diff
@@ -15,20 +17,38 @@ describe('GET Nomination Endpoint', () => {
   });
 
   it('returns a 200 when nomination exist', async () => {
-    const res = await request(app)
-      .get(`/nomination/${nomination.id}`);
+    const res = await request(app).get(`/nomination/${nomination.id}`);
     expect(res.statusCode).toBe(200);
   });
 
   it('return a 404 when nomination does not exist', async () => {
-    const res = await request(app)
-      .get('/nomination/00000000-0000-0000-0000-000000000000');
+    const res = await request(app).get(
+      '/nomination/00000000-0000-0000-0000-000000000000'
+    );
     expect(res.statusCode).toBe(404);
   });
 
   it('return a 400 when uuid is not a valid uuid', async () => {
-    const res = await request(app)
-      .get('/nomination/326');
+    const res = await request(app).get('/nomination/326');
     expect(res.statusCode).toBe(400);
   });
 });
+
+describe('Create Nomination Endpoint', () => {
```

Comment:
We may want a third test for the case where we pass a field with an invalid key. I would expect that to also return a 400 error code.